static/js/latest-news.js

```diff
@@ -0,0 +1,90 @@
+(function() {
+  function _slugify(string) {
+    const a = 'àáäâãåăæąçćčđďèéěėëêęğǵḧìíïîįłḿǹńňñòóöôœøṕŕřßşśšșťțùúüûǘůűūųẃẍÿýźžż·/_,:;'
+    const b = 'aaaaaaaaacccddeeeeeeegghiiiiilmnnnnooooooprrsssssttuuuuuuuuuwxyyzzz------'
+    const p = new RegExp(a.split('').join('|'), 'g')
+  
+    return string.toString().toLowerCase()
+      .replace(/\s+/g, '-') // Replace spaces with -
+      .replace(p, c => b.charAt(a.indexOf(c))) // Replace special characters
+      .replace(/&/g, '-and-') // Replace & with 'and'
+      .replace(/[^\w\-]+/g, '') // Remove all non-word characters
+      .replace(/\-\-+/g, '-') // Replace multiple - with single -
+      .replace(/^-+/, '') // Trim - from start of text
+      .replace(/-+$/, '') // Trim - from end of text
+  }
+
+  function _articleDiv(article, gtmEventLabel) {
+    const parsedDate = new Date(article.date);
+    var monthNames = [
+      "January", "February", "March",
+      "April", "May", "June", "July",
+      "August", "September", "October",
+      "November", "December"
+    ];
+    const formattedDate = parsedDate.getDate() + ' ' + monthNames[parsedDate.getMonth()] + ' ' + parsedDate.getFullYear();
+    const link = '<a href="/blog/' + article.slug + '">' + article.title.rendered + '</a>';
+    const header = "<h4>" + link + "</h4>";
+    const date = '<p class="u-no-padding--top"><em><time pubdate datetime="' + article.date + '">' + formattedDate + '</time></em></p>';
+    const div = document.createElement("div");
+    div.classList.add("col-3");
+    div.innerHTML = header + date;
+
+    div.querySelector('a').onclick = function() {
+      dataLayer.push(
+        {
+          'event': 'GAEvent',
+          'eventCategory': 'blog',
+          'eventAction': gtmEventLabel + ' news link',
+          'eventLabel': _slugify(article.title),
+          'eventValue': article.link
```

Comment:
eventValue is the numerical value of the click like $1. The label should cover the tracking.

Reply:
What should I do? Remove `eventValue`? Or return some property from the  `event` object?

Reply:
Oh just leave it `undefined`

Reply:
Ok done